decidim-participatory_processes/app/controllers/decidim/participatory_processes/admin/participatory_process_user_roles_controller.rb

```diff
@@ -45,7 +45,7 @@ def update
           enforce_permission_to :update, :process_user_role, process_user_role: @user_role
           @form = form(ParticipatoryProcessUserRoleForm).from_params(params)
 
-          UpdateParticipatoryProcessAdmin.call(@form, @user_role) do
+          UpdateParticipatoryProcessAdmin.call(@form, @user_role, current_participatory_process) do
```

Comment:
same here, it can be removed from the call: 

```suggestion
          UpdateParticipatoryProcessAdmin.call(@form, @user_role) do
```